docs/checks/message-translation.md

```diff
@@ -0,0 +1,35 @@
+[code pal for ABAP](../../README.md) > [Documentation](../check_documentation.md) > [Message Translation](message-translation.md)
+
+## Message Translation
+
+### What is the Intent of the Check?
+
+It identifies ´MESSAGES´ statements where the text is hard-coded as they cannot be translated.
```

Comment:
It identifies the ´MESSAGES´ statement where the text is hard-coded as it cannot be translated.